tests/backup/backup_upgrade_test.go

```diff
@@ -820,10 +993,15 @@ var _ = Describe("{PXBackupEndToEndBackupAndRestoreWithUpgrade}", Label(TestCase
 
 	})
 	JustAfterEach(func() {
-		allContexts := append(srcClusterContexts, destClusterContexts...)
-		defer EndPxBackupTorpedoTest(allContexts)
+		defer EndPxBackupTorpedoTest(partialAppContexts)
```

Comment:
who about having single instance of EndPxBackupTorpedoTest with allContexts

Reply:
Done, please review the changes.